test/prometheus_test.exs

```diff
@@ -374,8 +374,7 @@ defmodule PrometheusTest do
 
   defp lines_to_string(lines) do
     lines
-    |> Enum.intersperse(?\n)
-    |> then(&[&1, ?\n])
+    |> Enum.map(&[&1, ?\n])
```

Comment:
This was returning "\n" when there were no lines, if that's indeed the desired behaviour, please let me know.